api/agent/pure_runner.go

```diff
@@ -995,6 +997,19 @@ func PureRunnerWithStatusImage(imgName string) PureRunnerOption {
 	}
 }
 
+// PureRunnerWithKdumpsOnDisk returns a PureRunnerOption that indicates that
+// kdumps have been found on disk.  The argument numKdump is a counter that
+// indicates how many dumps were on disk at the time the runner was created.
+func PureRunnerWithKdumpsOnDisk(numKdumps uint64) PureRunnerOption {
+	return func(pr *pureRunner) error {
+		if pr.status.kdumpsOnDisk != 0 {
+			return fmt.Errorf("Duplicate kdump count configuration! old=%u new=%u", pr.status.kdumpsOnDisk, numKdumps)
```

Comment:
compiler complains about %d instead of %u here.

Reply:
Thanks for catching this; fixed.